From 607d0b42900940eae6d6ecbdf1eea9d9c837f75a Mon Sep 17 00:00:00 2001
From: Hiroshi Hatake <hatake@clear-code.com>
Date: Thu, 13 Jul 2017 16:41:23 +0900
Subject: [PATCH] Set ui.popup.disable_autohide as true to enable clicking
 events in popup menus on weston 
 0001-Fixed-rendering-of-noautohide-panels-rhbz-1466377.patch

---
 modules/libpref/init/all.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/libpref/init/all.js b/modules/libpref/init/all.js
index 6ee11a229c14..47ec1b17b339 100644
--- a/modules/libpref/init/all.js
+++ b/modules/libpref/init/all.js
@@ -233,7 +233,7 @@ pref("ui.click_hold_context_menus", false);
 // Duration of timeout of incremental search in menus (ms).  0 means infinite.
 pref("ui.menu.incremental_search.timeout", 1000);
 // If true, all popups won't hide automatically on blur
-pref("ui.popup.disable_autohide", false);
+pref("ui.popup.disable_autohide", true);
 
 pref("browser.display.use_document_fonts",  1);  // 0 = never, 1 = quick, 2 = always
 // 0 = default: always, except in high contrast mode
-- 
2.11.0

